frontend/src/mixins/FormatTimeLocal.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Vue } from "vue-property-decorator";
+import moment from "moment";
+
+@Component
+export default class FormatTimeLocal extends Vue {
+  formatTimeLocal(time: moment.MomentInput, format: string) {
```

Comment:
Currently you are using this function only in conjunction with `getDateFormat`, and the calls are looking non-intuitive and redundant. Like this: `formatTimeLocal(task.timestampGiven, getDateFormat(task.timestampGiven))`.

Just merge these two functions into `formatTimestamp` function that will always output the timestamp in the local format, date for past days, time for today.

One more idea: why not to have a single mixin for all timestamp/duration formatting functions? Wouldn't it be more intuitive than having many mixins? This is just a thought, I don't know the architecture and best practices of Vue, so you can refuse this proposal based on your experience.